src/ui/components/Overlay/styles.scss

```diff
@@ -1,5 +1,11 @@
 @import '~photon-colors/photon-colors';
 
+$z-index: (
```

Comment:
I think the idea was to have a variable that we can increment or that is the z-index of the overlay and then we do `$z-index + 1` and `$z-index + 2`.

Reply:
ahh! I see..thanks